src/apps/pfumain.cpp

```diff
@@ -0,0 +1,55 @@
+#include "common/Configuration.hpp"
+#include "handler.hpp"
+#include "pfu.hpp"
+
+#include <fmt/format.h>
+#include <lyra/lyra.hpp>
+#include <spdlog/sinks/stdout_color_sinks.h>
+#include <spdlog/spdlog.h>
+
+extern void boot_db();
```

Comment:
(non-actionable) one sweet day we'll be able to kill all these kinds of things. we should chat about how to start the process of OOficiation of the MUD. My interim so far has been to introduce (some) singletons, but I'm not sure how to proceed with minimal changes towards a hopeful inevitable "Mud" object comprising all the bits... anyway, one for chat sometime.